agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -241,4 +270,32 @@ private static final void setLasModifiedTime(NotificationInfo notification, Temp
                                                                      TimeConvertUtils.YEAR));
   }
 
+  private static final String getEventCreatorOrModifierUserName(long idUser) {
+    IdentityManager identityManager = ExoContainerContext.getService(IdentityManager.class);
```

Comment:
It's better to pass the dependent services in parameters of method instead of retrieving it from ExoContainerContext, see for example this method :
org.exoplatform.agenda.util.Utils.getIdentityById(long)